framework/MessageHandler.py

```diff
@@ -300,16 +300,15 @@ def error(self,caller,etype,message,tag='ERROR',verbosity='silent',color=None):
       @ Out, None
```

Comment:
As mentioned in the pull request, this is a side fix that came along the way as I was working.  

> Lastly, a small change was made to the MessageHandler to add a print statement with the error before raising the actual error, to tell which class is throwing the error and the time stamp.